### ISSUE TITLE
Crash (double free or corruption) with Python 3.6

### ISSUE THREAD

**Issue by JohanMabille** [jupyter-xeus/xeus-python#299]
The current master branch does not work with Python 3.6, starting th kernel leads to a crash:
```bash
*** Error in `xpython': double free or corruption (!prev): 0x000055762536a990 ***
```

This is due to the instantiation of [contextlib.nullcontext](https://github.com/jupyter-xeus/xeus-python/blob/master/src/xinteractiveshell.cpp#L29) which is not available before Python 3.7.

**Comment by SylvainCorlay** [comment]
Ping @btel FYI.

**Comment by SylvainCorlay** [comment]
I got a WIP fix in https://github.com/jupyter-xeus/xeus-python/pull/298.

Ping @martinRenou since you have done this kind of stuff for the completion.

**Comment by SylvainCorlay** [comment]
Fixed in #298
